src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2661,6 +2662,38 @@ NetworkOPsImp::getServerInfo(bool human, bool admin, bool counters)
         info["reporting"] = app_.getReportingETL().getInfo();
     }
 
+    // This array must be sorted in increasing order.
```

Comment:
Great question, and a good idea.  Since we're running C++20, `std::is_sorted()` is now `constexpr`.  So, if all of our standard libraries have that support, it should be easy to check at compile time.  Otherwise it's still possible to check, but a bit more code to write.

Reply:
Great idea. Thanks!